Sources/AWSSDKSwiftCore/AWSClient.swift

```diff
@@ -192,143 +181,149 @@ extension AWSClient {
 // public facing apis
 extension AWSClient {
 
-    /// send a request with an input object and return a future with an empty response
+    /// execute a request with an input object and return a future with an empty response
     /// - parameters:
     ///     - operationName: Name of the AWS operation
     ///     - path: path to append to endpoint URL
     ///     - httpMethod: HTTP method to use ("GET", "PUT", "PUSH" etc)
+    ///     - serviceConfig: configuration to be used to request creation and signing
```

Comment:
Maybe "AWS service configuration used in request creation and signing". Ha just noticed I hadn't put comments about the `eventLoop` and `stream` parameters in those functions. I'll do that once this is in.

Reply:
yeah that sounds alright, specifying that it is a config for AWS